perfkitbenchmarker/providers/gcp/gce_bigtable.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2014 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing class for GCP's bigtable clusters.
+
+Clusters can be created and deleted.
+"""
+
+import json
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import resource
+from perfkitbenchmarker.providers.gcp import util
+
+FLAGS = flags.FLAGS
+
+
+class GceBigtableCluster(resource.BaseResource):
+  """Object representing an GCE BigTable Cluster.
+
+  Attributes:
+    name: Cluster name.
+    num_nodes: Number of nodes in the cluster.
+    project: Enclosing project for the cluster.
+    zone: zone of the cluster.
+  """
+
+  def __init__(self, name, num_nodes, project, zone):
+    super(GceBigtableCluster, self).__init__()
+    self.num_nodes = num_nodes
+    self.name = name
+    self.zone = zone
+    self.project = project
+
```

Comment:
..and between other function definitions below.
https://google.github.io/styleguide/pyguide.html#Blank_Lines